src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -0,0 +1,295 @@
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Case sensitivity
+      </td>
+
+      <td>
+        * The data type names and attribute names are case sensitive.
+        * [NRQL clauses](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#clauses) and [functions](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#functions) are not case sensitive.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Syntax for strings
+      </td>
+
+      <td>
+        NRQL uses single quotes to designate strings. For example:
+
+        ```sql
+        ... where traceId = '030a573f0df02c57'
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Attribute names with spaces
+      </td>
+
+      <td>
+        Use backticks ``` `` ``` to quote a [custom attribute](/docs/insights/new-relic-insights/decorating-events/insights-custom-attributes) name that has a space in it. For example:
+
+        ```sql
+        ... FACET `Logged-in user`
+        ```
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Data type coercion
+      </td>
+
+      <td>
+        We don't support data type "coercion." For more information, see [Data type conversion](/docs/insights/nrql-new-relic-query-language/nrql-reference/nrql-syntax-components-functions#type-conversion).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Use of math functions
+      </td>
+
+      <td>
+        [Basic and advanced math functions](/docs/insights/nrql-new-relic-query-language/using-nrql/nrql-math-using-select) are supported in the [`SELECT`](/docs/insights/new-relic-insights/using-new-relic-query-language/nrql-reference#state-select) statement.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Subqueries
+      </td>
+
+      <td>
+        NRQL supports [subqueries](/docs/query-your-data/nrql-new-relic-query-language/get-started/subqueries-in-nrql).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `JOIN` clauses
+      </td>
+
+      <td>
+        NRQL does not have the equivalent of the SQL `JOIN` clause, but you can [simulate a `JOIN` with custom attributes](/docs/insights/new-relic-insights/using-new-relic-query-language/simulate-sql-join-functions-insights).
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+These are the basic rules for NRQL, and we have more information on [NRQL syntax and functions](/docs/query-data/nrql-new-relic-query-language/getting-started/nrql-syntax-components-functions) to help you build your queries.
+
+## Data exploration [#explore-data]
+
+One of the best ways to learn how to use NRQL is to access one of the [query tools](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language#where) and use it to explore your data. Here's an example of exploring your data using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder) and the suggested entries from the interface. 
```

Comment:
The repetition of "one" here is a little jarring. Not a big deal, but something to note.

Reply:
Agreed, changed it slightly to help make it smoother